generator/emitter_planet.cpp

```diff
@@ -11,13 +11,14 @@ using namespace std;
 
 namespace generator
 {
-EmitterPlanet::EmitterPlanet(feature::GenerateInfo const & info) :
+EmitterPlanet::EmitterPlanet(feature::GenerateInfo const & info, size_t numThreads) :
```

Comment:
Move `numThreas` into `GenerateInfo`. It makes no sense to transmit separately when a structure with parameters already comes here.